integration/test_flow_integration.py

```diff
@@ -928,7 +980,20 @@ def test_writing_timedelta_key(setup_teardown_test):
 
 
 def test_write_two_keys_to_v3io_from_df(setup_teardown_test):
-    table = Table(setup_teardown_test.table_name, setup_teardown_test.driver())
+    if setup_teardown_test.driver_name == "SQLDriver":
+        create_sql_table(
+            {"first_name": str, "last_name": str, "city": str},
+            setup_teardown_test._sql_table_name,
+            setup_teardown_test.sql_db_path,
+            ["first_name", "last_name"],
+        )
```

Comment:
also please see if you can reduce the boilerplate done in everytest:
create a Table constructor from a function and give that function sql_options, which will be used to call the create_sql_table if needed